Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -152,6 +155,19 @@ int RunVertices(int vtx_attr_group, int primitive, int count, DataReader src, bo
 	if (skip_drawing || is_preprocess)
 		return size;
 
+	// Hide Objects Code code
+	if (m_LocalCoreStartupParameter.num_object_removal_codes)
+	{
+		for (const SkipEntry& entry : m_LocalCoreStartupParameter.object_removal_codes)
+		{
+			if (!memcmp(src.GetPointer(), entry.data(), entry.size()))
```

Comment:
Can you please still try the performance of:
u64 hash = XXH64(src.GetPointer(), size, 0);
if (!memcmp(&hash, entry.data(), entry.size())) ...